.github/workflows/release.yml

```diff
@@ -35,7 +35,9 @@ jobs:
         run: npm run test
       - name: Coverage Report
         uses: codecov/codecov-action@v1
-      - name: Publish Changed Packages
-        run: npm run publish
+      - name: Push Updated Versions and Tags
+        run: lerna version --yes
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish Changed Packages
+        run: lerna publish from-package --yes
```

Comment:
`from-package` here should tell Lerna to get package versions from the package.json changes that were made in the previous step for `lerna version`, which are already committed to the rep at this point.